src/styles/_mixin.scss

```diff
@@ -0,0 +1,7 @@
+@mixin responsive ($map, $width) {
+  @each $size, $columns in $map {
+    @media (max-width: $size) {
+      grid-template-columns: repeat($columns, $width);
+    }
+  }
+}
```

Comment:
The use of this loop worsens the readability of the code. In order to fully comprehend what is happening, one must check the $media-query variable.

Remove it, and remove the mixin that you use once.